data/src/config/buffer.rs

```diff
@@ -1,13 +1,17 @@
-use serde::{Deserialize, Serialize};
```

Comment:
Lol `config::buffer` and `config::channel` files already existed but weren't previously modules. This must have been from a past refactor where we forgot to delete them :grimacing: